reliability-engineering/pipelines/concourse-deployer.yml

```diff
@@ -188,13 +188,15 @@ jobs:
   - do:
     - in_parallel:
       - task: scale-out-web-node-asg
-        file: tech-ops/reliability-engineering/pipelines/tasks/asg-scale-max-capacity.yml
+        file: tech-ops/reliability-engineering/pipelines/tasks/asg-scale-capacity.yml
         params:
           ASG_PREFIX: ((deployment_name))-concourse-web
+          SCALE_OUT: true
```

Comment:
This is a bit interesting - all it's possible to pass to tasks is environment variables (i.e. strings), but you're passing it the yaml boolean `true` here. It _just so happens_ that concourse converts that back into a lowercase `true`, which you're then doing a later exact comparison for so it works.

Maybe this is something we can rely on. Maybe we should try case-folding the comparison. Not sure. Just making sure you're aware of the oddness here.